### PR TITLE
add lave.js v1.1.9 with autoupdate config

### LINKED ISSUE #7323: [Request] Add lave
**Library name:** lave
**Git repository url:** https://github.com/jed/lave

PS: not sure if it works in browser or not, need to be confirmed.

### PR DESCRIPTION
**Pull request by dannyxx001**
Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/jed/lave
 * Official website (optional, not the repository): 
 * NPM package url (optional): https://www.npmjs.com/package/lave
 * GitHub / Bitbucket popularity (required):
   - Count of stars: 785
   - Count of watchers: 17
   - Count of forks: 17
 * NPM download stats (optional):
   - Downloads in the last day: 27
   - Downloads in the last week: 77
   - Downloads in the last month: 740

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [x] The lib has notable popularity
   * [x] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [x] More than 500 downloads stats per month on npm registry
 * [x] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [ ] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

close #7323, cc @jed

### CODE REVIEW

**Comment by dannyxx001** [comment]
@PeterDaveHello please help me review this pull request, thank you.

File: ajax/libs/lave/1.1.9/lave.min.js [hunk 1]
```diff
@@ -0,0 +1 @@
+// Error: Unexpected token: keyword (default)
```
Comment (PeterDaveHello) [inline ajax/libs/lave/1.1.9/lave.min.js]: Error.

File: ajax/libs/lave/package.json [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+    "json",
+    "serialize"
+  ],
+  "license": "MIT",
+  "npmName": "lave",
+  "npmFileMap": [
+    {
+      "basePath": "",
+      "files": [
+        "**/*"
```
Comment (PeterDaveHello) [inline ajax/libs/lave/package.json]: Should not use this kind of wildcard in non-dist folder/basePath.

Comment (PeterDaveHello) [inline ajax/libs/lave/package.json]: There are 6 files in latest npm ackage:
`index.js  lave.js  LICENSE  package.json  README.md  test.js`

So why to use `"**/*"` here? We don't need them all. Why not just `"lave*"`?

**Comment by dannyxx001** [comment]
@PeterDaveHello This is the website I found to minify the js files. (http://www.danstools.com/javascript-minify/) Original web-minify and http://jscompress.com/ and https://javascript-minifier.com/ are failed

File: ajax/libs/lave/1.1.9/lave.min.js [hunk 3]
```diff
@@ -0,0 +1 @@
...
let properties=[]
for(let name of names){let descriptor=Object.getOwnPropertyDescriptor(value,name)
if(Object(descriptor.value)!==descriptor.value)continue
if(map.has(descriptor.value))continue
let property={type:'Identifier',name}
if(object)property={type:'MemberExpression',object,property}
map.set(descriptor.value,property)
properties.push({value:descriptor.value,object:property})}
for(let property of properties){crawl(map,property.value,property.object)}
return map}
```
Comment (PeterDaveHello) [inline ajax/libs/lave/1.1.9/lave.min.js]: Still failed.

**Comment by dannyxx001** [comment]
@PeterDaveHello I have updated, please check again. Thanks.

**Pull request merged**